packages/react/src/i18n/translations.ts

```diff
@@ -98,7 +98,8 @@ export const Translations = {
             validationErrorMessage: 'This text area is invalid',
             maxLengthValidationErrorMessage:
                 'The number of characters exceeds the maximum allowed. You must shorten the description.',
-            characters: '{{current}}/{{max}} characters',
+            characters: '{{length}} / {{max}} characters',
+            charactersScreenReader: '{{length}} of {{max}} characters available',
```

Comment:
Je pense que le texte est à l'envers. Mettons qu'on a 18 chars sur 20 d'entrés, ça devrait dire `18 / 20 characters entered/used/autre synonyme` OU `2 / 20 characters available`. Personnellement je pense que la 1ère option a plus de sens dans le contexte d'un nombre de caractères.

Reply:
Au début je l'avais interprété comme "18 caractères entrés sur 20 caractères disponibles". P-e qu'en anglais, ça aurait dû être `18 / 20 available characters`?

Mais ta première suggestion laisse p-e moins de place à l'interprétation... 🤷

Reply:
Les textes ont ete fournis... je n'ai pas porte d'attention la dessus.  @LarryMatte 

P-t ca serait mieux de meme? 
```
19 characters (max limit of 100 characters)
19 characteres (limite maximale de 100 characteres)
```

Reply:
Je vérifie avec les designers qui focus sur le content et je vous reviens

Reply:
J'ai regardé avec Claudie et Laury. Leur conclusion serait de se limiter seulement à "xx/xx characters".

Reply:
En fait vous etes confu, les commentaires c'est plus pour le text du screen reader.

Reply:
Ahh je vois! On avait mal compris I guess. Voici ce qu'on propose afin d'être le plus explicite possible :
`charactersScreenReader: '{{length}} characters used of {{max}} characters available',`

Reply:
Question comme ça (j'ai jamais utilisé un screen reader): si le label est modifié à mesure que l'utilisateur entre du texte, est-ce que ça va ré-annoncer à chaque fois la phrase au complet?

Si c'est sur demande, c'est parfait comme ça! Si c'est imposé, c'est p-e un peu long comme texte! :)